Automattic/wp-calypso
# Plans: add plan features header component

This is part of the plans redesign, which breaks out the plan header component from @adambbecker's work in #5715 

![6ef5b45c-275d-11e6-895f-63de220aadf9](https://cloud.githubusercontent.com/assets/1270189/15878672/b6a11926-2cd1-11e6-8322-f88c322fbdcd.png)

This PR adds the plans header redesign to [devdocs](http://calypso.localhost:3000/devdocs/app-components/plan-features)
<img width="531" alt="screen shot 2016-06-09 at 10 54 58 am" src="https://cloud.githubusercontent.com/assets/1270189/15940633/bfd53fca-2e31-11e6-9288-1a25cbd348be.png">


## Testing Instructions
- No functional changes to Calypso.
- Navigate to http://calypso.localhost:3000/devdocs/app-components/plan
- You should see the PlanFeaturesHeader as shown above
- new selector tests pass:
`npm run test-client client/state/plans/test/selectors.js`

## todo
- [x] move svgs inline
- [x] use plan constants
- [x] add price plan selectors
- [x] handle price localization

cc @lamosty @rralian @mtias @rodrigoi
